sdk/inc/azure/core/az_context.h

```diff
@@ -41,6 +41,8 @@ struct az_context
     int64_t expiration; // Time when context expires
     void const* key; // Pointers to the key & value (usually NULL)
     void const* value;
+    // These are internal only fields and are subject to change without notice. They are not meant
```

Comment:
This comment should be at the start of the structure definition.  Ditto for all other locations

Reply:
`struct az_context
{ 1)
  struct
  { 2)
    az_context const* parent; // Pointer to parent context (or NULL); immutable after creation
    int64_t expiration; // Time when context expires
    void const* key; // Pointers to the key & value (usually NULL)
    void const* value;
    // These are internal only fields and are subject to change without notice. They are not meant to be used by anyone externally.
  } _internal;
};`

At location 1) or 2) ?

Reply:
1 is where we'd typically place the comment describing the structure.